client/purplecaffeine/core.py

```diff
@@ -187,18 +224,21 @@ def list(
         Returns:
             list of trials
         """
-        raise NotImplementedError
-
-    def get(self, name: str) -> "Trial":
-        """Returns trail by name.
+        offset = offset or 0
+        limit = limit or 10
+        trials = []
 
-        Args:
-            name: trail name
+        for to_get in range(offset, limit):
```

Comment:
With code above you will create list of requests from 0 to 10 :) 
We do not want to do that. We want to send one request with information on how many responses do we need back.

I think django should support offset/limit. Or maybe just `page` parameter in query string https://www.django-rest-framework.org/api-guide/pagination/#setting-the-pagination-style

so your request string should looks something like `http://bla-bla:8000/api/trials/&page=1`

if user want to get next batch he needs to call
```python
backend.list(page=2)
```
which will call for `http://bla-bla:8000/api/trials/&page=2`

Reply:
I've implemented limit/offset pagination for api server here https://github.com/IceKhan13/purplecaffeine/pull/39

Reply:
Yes page is working :)

Reply:
Right now I add your offset/limit call. Which one do you want to use ?